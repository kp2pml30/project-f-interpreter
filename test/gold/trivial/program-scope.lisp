(setq a 1)
(prog ()
    (setq a 2)
    (setq b 11)
    (print a)
    (print b))
(print b)
(print a)
(prog (a)
    (setq a 3)
    (print a))
(print a)
