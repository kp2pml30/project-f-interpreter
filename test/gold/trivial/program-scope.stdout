"int:2"
"int:11"
null
"int:2"
"int:3"
"int:2"
